superset-frontend/webpack.config.js

```diff
@@ -344,6 +344,11 @@ const config = {
         ],
         use: [babelLoader],
       },
+      {
```

Comment:
Please add a `todo` comment here and link to this [PR](https://github.com/apache/superset/issues/18655). We should refactor `DatasourceEditor.jsx` and decouple it in future.